src/Attachment.php

```diff
@@ -365,6 +367,26 @@ public function getExtension()
 
 
     /**
+     * Generate a temporary url at which the current file can be downloaded until $expire
+     *
+     * @param Carbon $expire
+     *
+     * @return string
+     */
+    public function getTemporaryUrl(Carbon $expire)
+    {
+
+        $payload = Crypt::encryptString(collect([
```

Comment:
is the payload guaranteed to be unique over time ? should it be better to prefix it with a timestamp or uniqid ?